doc/tang.8.adoc

```diff
@@ -135,6 +135,60 @@ of the database directory to all your servers. Make sure you don't forget the
 unadvertised keys! Then set up DNS round-robin so that clients will be load
 balanced across your servers.
 
+== AUTHORIZATION CONTROL
+
+By default, tang will respond to any recovery requests.  This can be
+controlled on a per client key by adding a second argument to the *tang*
+command line.  This argument need to be a path to an existing directory.
+This directory should be empty on first setup; this directory is called
+the 'authorization directory'.
+
+The concept is that an empty file named by the fingerprint of the client
+key needs to be present in the authorization directory to allow a
+decryption operation to happen.  If this file is missing, the request
+will be denied.
+
+An example:
+
+1. Configure tangd to use */var/db/tang/auth* as the authentication
+   directory.
+
+ifdef::freebsd[]
+   # TODO
```

Comment:
I will need some help here, how to best enable this on FreeBSD.

Reply:
The FreeBSD equivalent is really not much different (see units/tangd.rc.in where the tangd config is built from):

Alter the command_args line in the tangd file in the rc.d directory to be something like:
  command_args="${_tangd_listen_args} SYSTEM:\"${tangd_executable} ${tangd_jwkdir} /var/db/tang/authorized 2>> ${tangd_logfile} \" &"
and possibly add that directory to the required dirs line in the same file, i.e.,
required_dirs="${tangd_jwkdir} /var/db/tang/authorized"